### PR TITLE
similar behaviour moved into separate method

### PR DESCRIPTION
**Pull request by mkasztelnik**

### CODE REVIEW

**Comment by mkasztelnik** [comment]
@dice-cyfronet/atmo-dev-team please take a look

**Comment by mkasztelnik** [comment]
The inspiration was this report https://codeclimate.com/github/dice-cyfronet/atmosphere/Atmopshere::Clockwork

File: lib/atmosphere/clockwork.rb [hunk 1]
```diff
@@ -29,17 +25,27 @@ module Clockwork
         Atmosphere::BillingWorker.perform_async
       end
 
-      every(Atmosphere.url_monitoring.pending.seconds, 'monitoring.http_mappings.pending') do
+      every(Atmosphere.url_monitoring.pending.seconds,
+            'monitoring.http_mappings.pending') do
```
Comment (paoolo) [inline lib/atmosphere/clockwork.rb]: What you have changed here? Wrap lines?

Comment (mkasztelnik) [inline lib/atmosphere/clockwork.rb]: Here yes. The main difference is lines 7-8, 12-13 and 43-48. The rest is just formatting

Comment (paoolo) [inline lib/atmosphere/clockwork.rb]: OK. Looks cool.

**Comment by paoolo** [comment]
:+1:

**Comment by Nuanda** [comment]
:+1:

**Pull request merged**